.babelrc

```diff
@@ -0,0 +1,22 @@
+{
+  "presets": ["react", "es2015"],
```

Comment:
How familiar are you with these guys? I mean, react and es2015. Why choose them over other choices?